validator_manager/src/move_validators.rs

```diff
@@ -0,0 +1,1261 @@
+                    status,
+                ));
+            }
+            SingleExportKeystoresResponse {
+                status: Status { status, message },
+                ..
+            } => {
+                eprintln!(
+                    "Validator {:?} was not moved because the source validator client \
+                    indicated there was an error disabling it. Manual intervention is \
+                    required to recover from this scenario.",
+                    pubkey_to_move
+                );
+                return Err(format!(
+                    "VC returned status {:?} with message {:?}",
+                    status, message
+                ));
+            }
+        };
+
+        let keystore_derivation_path = voting_keystore.0.path();
+
+        let validator_specification = ValidatorSpecification {
+            voting_keystore,
+            voting_keystore_password,
+            slashing_protection: Some(InterchangeJsonStr(slashing_protection)),
+            fee_recipient,
+            gas_limit,
+            builder_proposals,
+            // Allow the VC to choose a default "enabled" state. Since "enabled" is not part of
+            // the standard API, leaving this as `None` means we are not forced to use the
+            // non-standard API.
+            enabled: None,
+        };
+
+        // We might as well just ignore validators that already exist on the destination machine,
+        // there doesn't appear to be much harm just adding them again and removing them from the
+        // source VC is an improvement.
+        let ignore_duplicates = true;
+
+        loop {
+            match validator_specification
+                .clone()
+                .upload(&dest_http_client, ignore_duplicates)
+                .await
+            {
+                Ok(status) => {
+                    match status.status {
+                        ImportKeystoreStatus::Imported => {
+                            eprintln!("Moved keystore {} of {}", i + 1, count);
+                            break;
+                        }
+                        ImportKeystoreStatus::Duplicate => {
+                            eprintln!("Moved duplicate keystore {} of {} to the VC", i + 1, count);
+                            break;
+                        }
+                        ImportKeystoreStatus::Error => {
+                            eprintln!(
+                                "Upload of keystore {} of {} failed with message: {:?}. \
+                                    A potential solution is run this command again \
+                                    using the --{} flag, however care should be taken to ensure \
+                                    that there are no duplicate deposits submitted.",
+                                i + 1,
+                                count,
+                                status.message,
+                                IGNORE_DUPLICATES_FLAG
+                            );
```

Comment:
We could remove the suggestion about duplicates, as the flag doesn't exist for the move command.